### PR TITLE
Updated Actual default values

### PR DESCRIPTION
**Pull request by Asizon**
This PR updates new Actual default values to go with https://github.com/betaflight/betaflight/pull/10724

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

File: src/js/tabs/pid_tuning.js [hunk 1]
```diff
@@ -2697,6 +2697,10 @@ TABS.pid_tuning.changeRatesSystem = function(sameType) {
                 rc_rate_pitch_e.val((FC.RC_TUNING.rcPitchRate * 1000).toFixed(0));
                 rc_rate_e.val((FC.RC_TUNING.RC_RATE * 1000).toFixed(0));
                 rc_rate_yaw_e.val((FC.RC_TUNING.rcYawRate * 1000).toFixed(0));
+            } else if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
+                rcRateDefault = (70).toFixed(0);
+                rateDefault = (670).toFixed(0);
+                expoDefault = (0).toFixed(2);
```
Comment (McGiverGim) [inline src/js/tabs/pid_tuning.js:2703]: We have default values in this file? I didn't notice it before. I thought all the defaults were at fc.js.

Comment (Asizon) [reply src/js/tabs/pid_tuning.js:2703]: yes it is designed like that in the @fgiudice98 original code for Rate types.

Comment (Asizon) [reply src/js/tabs/pid_tuning.js:2703]: Thougth that we never had Rate defaults in fc.js

Comment (McGiverGim) [reply src/js/tabs/pid_tuning.js:2703]: I don't like the way it is, but changing it now is outside of the scope of this PR, so to me is ok.

**Review by McGiverGim — APPROVED** [review]

File: src/js/tabs/pid_tuning.js (hunk 1 above)
Comment (Asizon) [reply src/js/tabs/pid_tuning.js:2703]: Maybe we can made this change for next PR's yes, i think it make sense.

**Review by mikeller — APPROVED** [review]

**Pull request merged**